### PR TITLE
Rust 1.28

### PR DESCRIPTION
**Pull request by Gottox**
This PR builds upon @jnbr's rust-1.27 update.

### CODE REVIEW

File: srcpkgs/rust/template [hunk 1]
```diff
@@ -19,18 +19,18 @@ maintainer="Enno Boland <gottox@voidlinux.eu>"
...
+checksum=9a818c50cdb7880abeaa68b3d97792711e6c64c1cdfb6efdc23f75b8ced0e15d
 
 case "$XBPS_MACHINE" in
 x86_64-musl)
 	hostmakedepends+=" libcurl libgit2"
 	distfiles+="
-	 https://repo.voidlinux.eu/distfiles/rustc-${_rust_dist_version}-x86_64-unknown-linux-musl.tar.gz
-	 https://repo.voidlinux.eu/distfiles/rust-std-${_rust_dist_version}-x86_64-unknown-linux-musl.tar.gz
+	 https://jnbr.me/distfiles/rustc-${_rust_dist_version}-x86_64-unknown-linux-musl.tar.xz
+	 https://jnbr.me/distfiles/rust-std-${_rust_dist_version}-x86_64-unknown-linux-musl.tar.xz
```
Comment (jnbr) [inline srcpkgs/rust/template]: please change it to repo.voidlinux.eu

Comment (Gottox) [reply srcpkgs/rust/template]: done

**Comment by jnbr** [comment]
`files/musl.patch` isn't used anymore and can be removed

**Comment by jnbr** [comment]
checksums for i686 are wrong

**Comment by jnbr** [comment]
had to (re-)add this patch to make x86_64-musl build succeed:
```
--- rustc-1.28.0-src/src/librustc_target/spec/x86_64_unknown_linux_musl.rs.orig	2018-08-08 23:21:51.673454236 +0200
+++ rustc-1.28.0-src/src/librustc_target/spec/x86_64_unknown_linux_musl.rs	2018-08-08 23:23:11.080834122 +0200
@@ -11,7 +11,7 @@
 use spec::{LinkerFlavor, Target, TargetResult};
 
 pub fn target() -> TargetResult {
-    let mut base = super::linux_musl_base::opts();
+    let mut base = super::linux_base::opts();
     base.cpu = "x86-64".to_string();
     base.max_atomic_width = Some(64);
     base.pre_link_args.get_mut(&LinkerFlavor::Gcc).unwrap().push("-m64".to_string());
```

**Comment by Gottox** [comment]
applied

**Pull request merged**